CONTRIBUTING.md

```diff
@@ -129,6 +129,40 @@ Run `rush build -t <packagename>` to build a single project, and all local proje
 
 By default, Rush only displays things written to `STDERR`. If you want to see the full output, pass `--verbose` to any of the build commands. This is also true for script commands (see below).
 
+### Testing
+
+Once the project has successfully built, you can test any of our packages by executing `rushx test` from the root of the package that you want to test. This will automatically run the tests in both NodeJS and in a browser. To target these environments individually, you can run `rushx test:node` and `rushx test:browser`.
+
+By default, these npm scripts run recorded tests. We use a custom recording library called `test-utils-recorder`. You can learn more about our recorder at the [test-utils-recorder's readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/README.md).
+
+#### Recorded tests
+
+To run unit tests, go to the root of the project you want to test and run `rushx unit-test`. These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses. You won't need to have any environment configuration, nor internet access, for these tests to run.
+
+Keep in mind that if you modify the test files or the underlying network calls of the project you're testing, the recordings will need to be re-generated.
+
+You can read more in the section: [Regenerating recordings](#regenerating-recordings).
+
+#### Live tests
+
+By default, the npm scripts `test`, `test:node` or `test:browser` run recorded tests. To run live tests, set the environment variable `TEST_MODE` to "live" and ensure you have the right Azure resources to test against.
```

Comment:
```suggestion
By default, the npm scripts `test`, `test:node` or `test:browser` run recorded tests. To run live tests, set the environment variable `TEST_MODE` to "live" and ensure you have the right Azure resources to test against. You can create these resources manually on your own or make use of the script we have for each service.
```

You can follow this up with the contents of the `Integration Testing with live services` section and remove the link you have a few lines below

Reply:
Unresolving as the contents of the `Integration Testing with live services` are still in a separate section instead of being pulled in here and combining with `Live Tests` section.

Also, I kept the `...` in the suggested edit for you to make the necessary edits to make the flow from the content above to the content in `Integration Testing with live services` that we should pull in

Reply:
I think I resolved this by mistake! Thank you